src/main/java/picard/sam/markduplicates/UmiAwareDuplicateSetIterator.java

```diff
@@ -38,12 +38,12 @@
 import htsjdk.samtools.DuplicateSetIterator;
 import htsjdk.samtools.SAMRecord;
 import htsjdk.samtools.util.CloseableIterator;
-import org.apache.commons.lang3.StringUtils;
 import picard.PicardException;
 
 import java.util.*;
 
 import static htsjdk.samtools.util.StringUtil.hammingDistance;
```

Comment:
i see that static imports have been used here before, but there seems to be a consensus against them...coudl you remove them please?